manager/testdata/new-blueprints/scenario2/plotter.yaml

```diff
@@ -0,0 +1,167 @@
+apiVersion: app.m4d.ibm.com/v1alpha2
+kind: Plotter
+metadata:
+  name: app1-ns1 # plotter name is a combination of app and namespace
+  namespace: m4d-system
+  labels:
+spec:
+  appSelector: # Selector of the application that uses this workload
+    clusterName: thegreendragon
+    workloadSelector:
+      matchLabels:
+        app: demoapp
+  assets:
+  - dataSetID: "m4d-notebook-sample/paysim"
+    source:
+      credentials:
+        read:
+          vault:
+            address: http://vault.m4d-system:8200
+            authPath: /v1/auth/kubernetes/login
+            role: module
+            secretPath: /v1/kubernetes-secrets/paysim?namespace=m4d-notebook-sample
+      connection:
+        s3:
+          endpoint: localhost:8001
+          bucket: srcbucket
+          object: data.parq
+      dataformat: parquet
+  - dataSetID: "m4d-notebook-sample/users"
+    source:
+      credentials:
+        read:
+          vault:
+            address: http://vault.m4d-system:8200
+            authPath: /v1/auth/kubernetes/login
+            role: module
+            secretPath: /v1/kubernetes-secrets/users?namespace=m4d-notebook-sample
+      connection:
+        s3:
+          endpoint: localhost:8001
+          bucket: srcbucket
+          object: users.parq
+      dataformat: parquet
+  - dataSetID: "m4d-notebook-sample/inventory"
+    source:
+      credentials:
+        read:
+          vault:
+            address: http://vault.m4d-system:8200
+            authPath: /v1/auth/kubernetes/login
+            role: module
+            secretPath: /v1/kubernetes-secrets/inventory?namespace=m4d-notebook-sample
+      connection:
+        s3:
+          endpoint: localhost:8001
+          bucket: srcbucket
+          object: inventory.parq
+      dataformat: parquet
+  flows:
+  - name: flow-paysim
+    flowType: read
+    dataSetID: "m4d-notebook-sample/paysim"
+    subflows:
```

Comment:
@froesef In the document we had a type for the sub-flow as well.  I'm looking at scenario #6 and I think we need it for that.